AsaApi/Core/Public/API/ARK/Inventory.h

```diff
@@ -969,7 +969,7 @@ struct UPrimalInventoryComponent : UActorComponent
     bool LoadAdditionalStructureEngrams() { return NativeCall<bool>(this, "UPrimalInventoryComponent.LoadAdditionalStructureEngrams()"); }
     bool RemoveItem(const FItemNetID* itemID, bool bDoDrop, bool bSecondryAction, bool bForceRemoval, bool showHUDMessage) { return NativeCall<bool, const FItemNetID*, bool, bool, bool, bool>(this, "UPrimalInventoryComponent.RemoveItem(FItemNetID&,bool,bool,bool,bool)", itemID, bDoDrop, bSecondryAction, bForceRemoval, showHUDMessage); }
     ADroppedItem* EjectItem(const FItemNetID* itemID, bool bPreventImpule, bool bForceEject, bool bSetItemLocation, const UE::Math::TVector<double>* LocationOverride, bool showHUDMessage, TSubclassOf<ADroppedItem> TheDroppedTemplateOverride, bool bAssignToTribeForPickup, int AssignedTribeID) { return NativeCall<ADroppedItem*, const FItemNetID*, bool, bool, bool, const UE::Math::TVector<double>*, bool, TSubclassOf<ADroppedItem>, bool, int>(this, "UPrimalInventoryComponent.EjectItem(FItemNetID*,bool,bool,bool,UE::Math::TVector<double>*,bool,TSubclassOf<ADroppedItem>,bool,int)", itemID, bPreventImpule, bForceEject, bSetItemLocation, LocationOverride, showHUDMessage, TheDroppedTemplateOverride, bAssignToTribeForPickup, AssignedTribeID); }
-    bool ServerEquipItem(FItemNetID& itemID) { return NativeCall<bool, FItemNetID&>(this, "UPrimalInventoryComponent.ServerEquipItem(FItemNetID&)", itemID); }
+    bool ServerEquipItem(FItemNetID* itemID, AShooterPlayerController* RequestSPC) { return NativeCall<bool, FItemNetID*, AShooterPlayerController*>(this, "UPrimalInventoryComponent.ServerEquipItem(FItemNetID&,AShooterPlayerController*)", itemID, RequestSPC); }
```

Comment:
This is a duplicate of https://github.com/ArkServerApi/AsaApi/pull/16

Please remove this.